src/libraries/System.Private.CoreLib/src/System/Runtime/Versioning/PlatformAttributes.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.Versioning
+{
+    /// <summary>
+    /// Base type for all platform-specific API attributes.
+    /// </summary>
+#pragma warning disable CS3015 // Type has no accessible constructors which use only CLS-compliant types
+#if INTERNAL_PLATFORM_ATTRIBUTES
```

Comment:
This should use existing `SYSTEM_PRIVATE_CORELIB` define.
```
#if SYSTEM_PRIVATE_CORELIB
public
#else
internal
#endif
``` 
There are number of other attributes on the same plan.

Reply:
(You can fix `INTERNAL_NULLABLE_ATTRIBUTES` to also use `SYSTEM_PRIVATE_CORELIB` while you are on it.)

Reply:
done